middleware/locale/locale.go

```diff
@@ -0,0 +1,56 @@
+package locale
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/mholt/caddy/middleware"
+	"github.com/mholt/caddy/middleware/locale/method"
```

Comment:
Ah - maybe I should say that I consider the package name part of the type name. So instead of `HeaderMethod` I usually write `method.Header`. The same goes for functions on packages and functions on receiver types.